11.0/Dockerfile

```diff
@@ -73,12 +73,26 @@ RUN set -x; \
         && rm -rf /var/lib/apt/lists/* /root/.cache/pip/*
 
 # grab gosu for easy step-down from root
- RUN gpg --keyserver pool.sks-keyservers.net --recv-keys B42F6819007F00F88E364FD4036A9C25BF357DD4 \
-         && curl -o /usr/local/bin/gosu -SL "https://github.com/tianon/gosu/releases/download/1.10/gosu-$(dpkg --print-architecture)" \
-         && curl -o /usr/local/bin/gosu.asc -SL "https://github.com/tianon/gosu/releases/download/1.10/gosu-$(dpkg --print-architecture).asc" \
-         && gpg --verify /usr/local/bin/gosu.asc \
-         && rm /usr/local/bin/gosu.asc \
-         && chmod +x /usr/local/bin/gosu
+# fix intermittent gpg server error from https://github.com/tianon/gosu/issues/35
+RUN export GNUPGHOME="$(mktemp -d)" \
```

Comment:
maybe you can put this into a separated script to not pollute the Dockerfile?

Reply:
And it's so painful to write "one-line" bash code with `\` and `&&` :laughing: